client/src/components/NavBar.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { Link, Redirect } from "react-router-dom";
+import { useState, useEffect } from "react";
+import { Restaurant } from "./Restaurant";
```

Comment:
I noticed there is the occasional dependency that is faded-out, signaling they are not being used anywhere. For instance, this Restaurant component on line 4 is imported but never used in this file. It may be useful / good practice to go through and clean up any loose ends!